styles/globals.css

```diff
@@ -1,3 +1,4 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+@import './typography.css';
```

Comment:
Typography import should be wrapped in it's appropriate layer so it won't override tailwind utilities or at least change the order to be before tailwind utilities